Makefile

```diff
@@ -55,15 +55,12 @@ deps: tools
 tools:
 	@go get github.com/golang/dep/cmd/dep
 
-lint: prototool-bin
-	prototool lint
-
-prototool-bin:
+lint:
 ifndef $(shell command -v prototool help > /dev/null)
-	@@curl -sSL https://github.com/uber/prototool/releases/download/v1.3.0/prototool-$(shell uname -s)-$(shell uname -m) -o $(HOME)/.bin/prototool
-	@@chmod +x $(HOME)/.bin/prototool
+	@go get github.com/uber/prototool/cmd/prototool
+	@go install github.com/uber/prototool/cmd/prototool
```

Comment:
Doesn't go get also install the item if pointed to a main package and there is no `-d` flag?

I mean, it is nice how is, clearer and "safer", mainly just checking my understanding of the tools (been some months)

Reply:
@ethanfrey is right. The install line should be obsolete

> Get downloads the packages named by the import paths, along with their dependencies. It then installs the named packages, like 'go install'.

https://golang.org/cmd/go/#hdr-Download_and_install_packages_and_dependencies